File: JavaSE/Day 29/shop/apple/AdminMenu.java
package shop.apple;

import java.util.Scanner;

public class AdminMenu {

    void displayMenu(){
        System.out.println("=".repeat(25));
        System.out.println( " \uD83C\uDF4E Admin Dashboard");
        System.out.println("=".repeat(25));
        System.out.println("(1) Manage Products");
        System.out.println("(2) View Customer Accounts");
        System.out.println("(3) View Transactions");
        System.out.println("(0) Logout");
    }
    public  void options(){
        Scanner scanner = new Scanner(System.in);


        boolean aMenuRunning = true;
        while (aMenuRunning){
            //Calls Admin display
            displayMenu();

            System.out.print("To get start, select a number: ");
            int number = scanner.nextInt();
            System.out.println();

            switch (number){
                case 1:
                    new ManageProducts().displayPMenu();
                    break;
                case 2:
                    System.out.println("----------------------");
                    System.out.println("|  Customer Accounts  |");
                    System.out.println("----------------------");
                   Customer.viewCustomer();
                    break;
                case 3:
                    System.out.println("----------------------");
                    System.out.println("|    Transactions     |");
                    System.out.println("----------------------");
                    //new Transactions().viewAllTransactions();
                    break;
                case 0:
                    System.err.println("Logout out...");
                    aMenuRunning = false;
                    //Return to
                    new Login().beginLogin();
                    break;
                default:
                    System.out.println("Invalid option, Try again.");
            }
        }

        scanner.close();

    }


}

File: Android/Dart/shop/apple/AdminMenu.dart
import 'dart:io';

import 'Customer.dart';
import 'Login.dart';
import 'ManageProducts.dart';

class Adminmenu {
  static void showMenu() {
    print("=" * 25);
    print(" \uD83c\uDF4E Admin Dashboard");
    print("=" * 25);
    print(
      "(1) Manage Products\n(2) View Customer Accounts\n(3) View Transactions\n(0) Logout",
    );
  }

  static void options() {
    bool aMenuRunning = true;

    while (aMenuRunning) {
      showMenu();

      stdout.write("To get started, select a number: ");
      String? input = stdin.readLineSync();
      print("\n");

      int? number = int.tryParse(input!);

      switch (number) {
        case 1:
          //Send Admin to Product Management Panel
          print("Manage Prodcts show Menue...");
          new Manageproducts().showPMemenu();
          break;

        case 2:
          print("----------------------------");
          print("|     Customer Accounts    |");
          print("----------------------------");
          print("Customer View Customer...");
          print("\n");
          Customer.viewCustomers();
          break;

        case 3:
          print("----------------------------");
          print("|       Transactions       |");
          print("----------------------------");
          print("Transactions View All Transactions...");
          print("\n");
          break;

        case 0:
          stderr.writeln("Loggin out...");
          aMenuRunning = false;
          Login.beginLogin();
          break;
        default:
          print("Invalid option. Try agin.");
      }
    }
  }
}
